client/src/containers/case-studies/list/component.tsx

```diff
@@ -0,0 +1,36 @@
+import { useCaseStudies } from 'hooks/case-studies';
+
+import CaseStudiesListItem from '../list-item/component';
+
+const NUM_LOADING_SKELETON_ENTRIES = 6;
+
+const CaseStudiesList = () => {
+  const { data: caseStudies, isFetching, isFetched, isError } = useCaseStudies();
+
+  return (
+    <section className="space-y-4 py-10">
+      {isError ? (
```

Comment:
Try to avoid ternaries. Check this component to see how you can deal with that => https://github.com/Vizzuality/foodscapes/blob/develop/client/src/containers/widget/content/index.tsx#L25-L37